ethers-ffi/Makefile

```diff
@@ -40,37 +40,31 @@ android: android-setup android-build
 	cp $(TARGET_DIR)/i686-linux-android/release/$(SO) ${ANDROID_DEST}/x86/$(SO)
 	cp $(TARGET_DIR)/x86_64-linux-android/release/$(SO) ${ANDROID_DEST}/x86_64/$(SO)
 	cp $(TARGET_DIR)/aarch64-linux-android/release/$(SO) ${ANDROID_DEST}/arm64-v8a/$(SO)
-	cp $(TARGET_DIR)/arm-linux-androideabi/release/$(SO) ${ANDROID_DEST}/armeabi/$(SO)
+
 	cp $(TARGET_DIR)/armv7-linux-androideabi/release/$(SO) ${ANDROID_DEST}/armeabi-v7a/$(SO)
 
 aarch64-linux-android:
 	PATH=$(PATH):$(NDK_STANDALONE)/arm64/bin \
-	CC=$@-gcc \
-	CXX=$@-g++ \
-	cargo build $(CARGO_PARAMS) --target $@ --release --lib
-
-arm-linux-androideabi:
-	PATH=$(PATH):$(NDK_STANDALONE)/arm/bin \
-	CC=$@-gcc \
-	CXX=$@-g++ \
+	CC=$@-clang \
+	CXX=$@-clang++ \
 	cargo build $(CARGO_PARAMS) --target $@ --release --lib
 
 armv7-linux-androideabi:
 	PATH=$(PATH):$(NDK_STANDALONE)/arm/bin \
-	CC=arm-linux-androideabi-gcc \
-	CXX=arm-linux-androideabi-g++ \
+	CC=armv7a-linux-androideabi-gcc \
```

Comment:
Can't say i'm entirely sure why the target needs to be `arm7` but the toolchain file is `arm7a`. Tried a few permutations of namign between the `android-setup` and `android-build` naming and this was the one that worked. 

FWIW, the prebuilt LLVM NDK does not contain anything for `arm7` architecture.